values.yaml

```diff
@@ -30,12 +31,21 @@ client:
   # It isn't common to need more than 2 client nodes.
   replicas: 2
   antiAffinity: "soft"
+
+  # The amount of RAM allocated to the JVM heap. This should be set to the
+  # same value as resources.requests.memory, or you may see OutOfMemoryErrors
```

Comment:
Might be nitpicking, but shouldn't we put the full path here? `client.resources.requests.memory`

Reply:
Updated.